projects/igniteui-angular/src/lib/tree/tree-node/tree-node.component.ts

```diff
@@ -128,6 +139,18 @@ export class IgxTreeNodeComponent<T> extends ToggleAnimationPlayer implements Ig
         };
     }
 
+    /**
+     * The native DOM element representing the node. Could be null in certain environments.
+     *
+     * ```typescript
+     * // get the nativeElement of the second node
+     * let nodeNativeElement = this.tree.nodes.toArray()[1].nativeElement;
```

Comment:
Change this a bit, so it emphasizes on the property
```suggestion
     * const node: IgxTreeNode = this.tree.nodes.first();
     * const nodeElement: HTMLElement = node.nativeElement;
```